agent_api_rest/src/issuance/credential_issuer/credential.rs

```diff
@@ -283,6 +283,7 @@ mod tests {
 
             let target_url = format!("{}/ssi-events-subscriber", &external_server.uri());
 
+            set_config().credential_offer_by_value_enabled = Some(true);
```

Comment:
Setting the variable like this won't be necessary after you set it directly in the `test-config.yaml` file (see comment above).